### ISSUE TITLE
[FEATURE] Add label to multichannel mode

### ISSUE THREAD

**Issue by sgofferj** [charlie-foxtrot/RTLSDR-Airband#434]
**Is your feature request related to a problem? If so, please describe.**
Currently there's only frequency labels in scan mode which makes somewhat sense, however, labels would be useful also for multichannel mode, especially for Prometheus data and when using mixers.

**Describe the solution you'd like**
Add a "label" directive to channel definitions in multichannel mode. Use this label in Prometheus statistics and send it to Icecast as "Now playing" metadata.

**Describe alternative solutions or features you've considered**
No alternatives exist.

**Additional context**
n/a

**Comment by charlie-foxtrot** [comment]
yep, `label` is already supported in multi-channel mode but looks like the documentation is missing.  Will keep this open until the wiki is updated.
